Unable to create schedule

When trying to create a schedule for notebook, nothing happens. 
Developer tools in chrome gives below error
```
bundle.js:8 POST http://localhost:8080/api/v1/jobs?action=save 500 (Internal Server Error)
(anonymous) @ bundle.js:8
t.requestFormData @ bundle.js:8
(anonymous) @ bundle.js:8
n.onsubmit @ bundle.js:8
bundle.js:8 Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at Object.json (bundle.js:8)
    at bundle.js:8

```
Logs:
```
[2019-05-08 09:28:11 +0000] [10] [INFO] Booting worker with pid: 10
CRITICAL:root:Storing user admin eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6IjEiLCJuYW1lIjoiYWRtaW4ifQ.aS8njo_SfV2FlfoJe1w-5BVR23o8s7vWZYEuYWx6L8M 1
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.notebook.NotebookSQL'>, result : 0 - 0
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.job.JobSQL'>, result : 0 - 0
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.report.ReportSQL'>, result : 0 - 0
CRITICAL:root:Storing notebook <Notebook(name='Test', user='<User(name='admin')>', privacy='public', level='production'>
CRITICAL:root:Storing job <Job(name='Test Job', user='<User(name='admin')>', notebook='<Notebook(name='Test', user='<User(name='admin')>', privacy='public', level='production'>', start='2019-05-08 09:28:00', interval='hourly'>
[2019-05-08 09:29:08 +0000] [10] [ERROR] Error handling request /api/v1/jobs?action=save
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/gunicorn-19.9.0-py3.6.egg/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.6/dist-packages/gunicorn-19.9.0-py3.6.egg/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "falcon/api.py", line 274, in falcon.api.API.__call__
  File "falcon/api.py", line 269, in falcon.api.API.__call__
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/resources/job.py", line 19, in on_post
    resp.body = json.dumps(self.db.jobs.store(req.context['user'], req.params, self.session, self.scheduler))
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/storage/sqla/job.py", line 79, in store
    report_configs = self.db.reports.generate(user, params, session, job=job_config, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/storage/sqla/report.py", line 102, in generate
    report_params = parameters.file.read().decode('utf-8')
AttributeError: 'bytes' object has no attribute 'file'
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.job.JobSQL'>, result : 0 - 0
CRITICAL:root:Storing job <Job(name='Test Job', user='<User(name='admin')>', notebook='<Notebook(name='Test', user='<User(name='admin')>', privacy='public', level='production'>', start='2019-05-08 09:28:00', interval='hourly'>
[2019-05-08 09:32:42 +0000] [10] [ERROR] Error handling request /api/v1/jobs?action=save
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/gunicorn-19.9.0-py3.6.egg/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.6/dist-packages/gunicorn-19.9.0-py3.6.egg/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "falcon/api.py", line 274, in falcon.api.API.__call__
  File "falcon/api.py", line 269, in falcon.api.API.__call__
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/resources/job.py", line 19, in on_post
    resp.body = json.dumps(self.db.jobs.store(req.context['user'], req.params, self.session, self.scheduler))
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/storage/sqla/job.py", line 79, in store
    report_configs = self.db.reports.generate(user, params, session, job=job_config, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/storage/sqla/report.py", line 102, in generate
    report_params = parameters.file.read().decode('utf-8')
AttributeError: 'bytes' object has no attribute 'file'
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.job.JobSQL'>, result : 0 - 0
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.notebook.NotebookSQL'>, result : 1 - 1
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.job.JobSQL'>, result : 0 - 0
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.report.ReportSQL'>, result : 0 - 0
CRITICAL:root:Storing job <Job(name='5tel', user='<User(name='admin')>', notebook='<Notebook(name='Test', user='<User(name='admin')>', privacy='public', level='production'>', start='2019-05-08 09:32:00', interval='hourly'>
[2019-05-08 09:33:18 +0000] [10] [ERROR] Error handling request /api/v1/jobs?action=save
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/gunicorn-19.9.0-py3.6.egg/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.6/dist-packages/gunicorn-19.9.0-py3.6.egg/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "falcon/api.py", line 274, in falcon.api.API.__call__
  File "falcon/api.py", line 269, in falcon.api.API.__call__
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/resources/job.py", line 19, in on_post
    resp.body = json.dumps(self.db.jobs.store(req.context['user'], req.params, self.session, self.scheduler))
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/storage/sqla/job.py", line 79, in store
    report_configs = self.db.reports.generate(user, params, session, job=job_config, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/paperboy-0.0.2-py3.6.egg/paperboy/storage/sqla/report.py", line 102, in generate
    report_params = parameters.file.read().decode('utf-8')
AttributeError: 'bytes' object has no attribute 'file'
CRITICAL:root:list : <class 'paperboy.storage.sqla.models.job.JobSQL'>, result : 0 - 0
```

**To Reproduce**
Steps to reproduce the behavior:
1. Add -> Schedule
2. Provide details - schedule name, notebook, date at default, set interval to hourly, Level production, every thing else at default. 
3. Click save

**Expected behavior**
Scheduler gets saved

**Installation notes**
 - Container OS - Ubuntu. Installed dependencies then ran make install. CMD - python3 -m paperboy.server
 -  Python version - 3.7
 - Client - Mac os
 - Browser - Chrome 74.0.3729.131 (Official Build) (64-bit)

--- a/paperboy/scheduler/airflow/airflow.py
+++ b/paperboy/scheduler/airflow/airflow.py
@@ -148,6 +148,7 @@ class AirflowScheduler(BaseScheduler):
         '''Schedule a DAG for `job` composed of `reports` to be run on airflow'''
         template = AirflowScheduler.template(self.config, user, notebook, job, reports, *args, **kwargs)
         name = job.id + '.py'
+        os.makedirs(self.config.scheduler.dagbag, exist_ok=True)
         with open(os.path.join(self.config.scheduler.dagbag, name), 'w') as fp:
             fp.write(template)
         return template

--- a/paperboy/storage/sqla/report.py
+++ b/paperboy/storage/sqla/report.py
@@ -101,9 +101,8 @@ class ReportSQLStorage(BaseSQLStorageMixin, ReportStorage):
             parameters_inline = params.get('parameters_inline') or ''
             parameters = params.get('parameters')
 
-            if not parameters_inline:
-                if parameters is not None:
-                    report_params = parameters.file.read().decode('utf-8')
+            if not parameters_inline and parameters:
+                report_params = parameters.file.read().decode('utf-8')
             else:
                 report_params = parameters_inline
             if not report_params:

